### PR TITLE
fix search engine

### PR DESCRIPTION
**Pull request by Siedemus**

### CODE REVIEW

**Review by mileni67 — APPROVED** [review]

**Comment by DominikaBuczekSzkowron** [comment]
good job! can you correct this cases as well?

2023-08-26 uwagi - szukajka

2023-08-26 uwagi2 - szukajka

File: src/Features/Navigation/Search/index.js [hunk 1]
```diff
@@ -1,49 +1,35 @@
 import { Wrapper, Input } from "./styled";
 import { useDispatch, useSelector } from "react-redux";
```
Comment (DominikaBuczekSzkowron) [inline src/Features/Navigation/Search/index.js:2]: 'useSelector' is defined but never used

File: src/Features/Navigation/Search/index.js [hunk 2]
```diff
@@ -1,49 +1,35 @@
...
-import { onSearchChange, selectQuery } from "./searchSlice";
-import { fetchMovies } from "../../MoviesPage/moviesSlice";
-import { fetchPeopleListLoad } from "../../PeoplePage/peopleSlice";
-import { useManualSearchParamChange } from "./useManualSearchParamChange";
+import { setQuery, selectQuery } from "./searchSlice";
```
Comment (DominikaBuczekSzkowron) [inline src/Features/Navigation/Search/index.js:3]: 'selectQuery' is defined but never used

File: src/Features/Navigation/Search/useReplaceSearchParam.js [hunk 3]
```diff
@@ -1,31 +1,36 @@
 import { useEffect } from "react";
```
Comment (DominikaBuczekSzkowron) [inline src/Features/Navigation/Search/useReplaceSearchParam.js:1]: 'useEffect' is defined but never used

**Pull request merged**